### PR TITLE
Friend add frontend

### PR DESCRIPTION
**Pull request by Eneshjakhar**
Added: 
-FriendAdd.html
-FriendAdd.css
(Based on FriendAdd.js)

-minor changes to home.html
-minor changes to home.css

### CODE REVIEW

File: DiscordCode/BackEnd/routes/FriendAdd.js
Comment (ez4papiz1) [inline DiscordCode/BackEnd/routes/FriendAdd.js:1]: Looks good

File: DiscordCode/FrontEnd/FriendAdd.html
Comment (ez4papiz1) [inline DiscordCode/FrontEnd/FriendAdd.html:1]: Looks good

File: DiscordCode/FrontEnd/css/FriendAdd.css
Comment (ez4papiz1) [inline DiscordCode/FrontEnd/css/FriendAdd.css:1]: Looks good

**Review by ez4papiz1 — COMMENTED** [review]
Changes look correct. Should be able to merge without conflict

**Review by Lunilux24 — APPROVED** [review]
Looks good. Will merge now.

**Pull request merged**